maplibre/src/render/camera.rs

```diff
@@ -276,6 +276,10 @@ impl Camera {
         &self,
         inverted_view_proj: &InvertedViewProjection,
     ) -> Option<Aabb2<f64>> {
+        if self.width == 0. || self.height == 0. {
+            return None;
+        }
```

Comment:
Is this required?

Reply:
Otherwise `window_to_world_at_ground` -> `window_to_world` crashes at `self.clip_to_window_transform().invert().unwrap()`.

Reply:
I ensured somwhere else that these are always non-zero.